config/samples/operator_v1_antreainstall.yaml

```diff
@@ -543,6 +543,6 @@ spec:
     multicluster:
       # Enable Multi-cluster NetworkPolicy.
       enableStretchedNetworkPolicy: false
-  antreaImage: antrea/antrea-ubuntu:v1.14.0
-  antreaPlatform: kubernetes
+  antreaImage: antrea/antrea-ubi:v1.14.1
+  antreaPlatform: openshift
```

Comment:
a similar comment: by default, running `make manifests` will use `ubuntu` / `kubernetes`. I am not sure what the expectation is for the checked-in file..

Reply:
The process of generation and release is described [here](https://github.com/vmware/antrea-operator-for-kubernetes/blob/main/doc/release-process.md).
It's a bit of a mess, and can definitely be improved.